kedro/config/omegaconf_config.py

```diff
@@ -178,9 +186,15 @@ def __getitem__(self, key) -> dict[str, Any]:
             base_path = str(Path(self.conf_source) / self.base_env)
         else:
             base_path = str(Path(self._fs.ls("", detail=False)[-1]) / self.base_env)
-        base_config = self.load_and_merge_dir_config(
-            base_path, patterns, key, processed_files, read_environment_variables
-        )
+        try:
+            base_config = self.load_and_merge_dir_config(
+                base_path, patterns, key, processed_files, read_environment_variables
+            )
+        except UnsupportedInterpolationType:
+            raise UnsupportedInterpolationType(
+                "The `runtime_params:` resolver is not supported for globals."
+            )
```

Comment:
This could potentially be raised because of something else right? So we should make sure that this specific message only shows when the user is trying to use `runtime_params` with globals.

Reply:
I've updated the condition to only show this message if `runtime_params` is used.